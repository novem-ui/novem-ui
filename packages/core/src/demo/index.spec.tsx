import React from 'react'

import { render } from '@testing-library/react'

import DemoComponent from '.'

test('demo component', () => {
  const { container } = render(<DemoComponent text="demo" />)

  expect(container).toMatchSnapshot();
})
