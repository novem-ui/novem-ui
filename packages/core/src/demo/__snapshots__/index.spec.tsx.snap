// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`demo component 1`] = `
<div>
  <div>
    demo
  </div>
</div>
`;
